accounts-db/src/read_only_accounts_cache.rs

```diff
@@ -57,17 +59,33 @@ impl ReadOnlyCacheStats {
         self.load_us.store(0, Ordering::Relaxed);
         self.store_us.store(0, Ordering::Relaxed);
         self.evict_us.store(0, Ordering::Relaxed);
+        self.evictor_wakeup_count_all.store(0, Ordering::Relaxed);
+        self.evictor_wakeup_count_productive
+            .store(0, Ordering::Relaxed);
     }
 
-    fn get_and_reset_stats(&self) -> (u64, u64, u64, u64, u64, u64) {
+    fn get_and_reset_stats(&self) -> (u64, u64, u64, u64, u64, u64, u64, u64) {
```

Comment:
nit: The return tuple is growing large. Maybe it is time to change it into a struct?

Reply:
yeah.